.eslintrc.js

```diff
@@ -1,3 +1,12 @@
+// React is installed in src/web but Eslint is unable to detect it
+const { react } = require('./src/web/package.json').dependencies;
+
+// Extract react version manually to pass it to Eslint
+// Semver Regex: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+const reactVersion = react.match(
+  /(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
```

Comment:
This is horrible.  Can we just not check the React version?

Reply:
I can just hard-code React v17.0.

Reply:
Let's do that and fix it when we upgrade React later